src/main/twirl/views/UploadGuiPage.scala.html

```diff
@@ -62,119 +76,164 @@
+						</div>
+						<div class="form-group">
+							<label>Number of rows <em class="text-muted">- optional</em></label>
+							<input id="nrows" type="text" class="form-element form-control" min="0" disabled>
 						</div>
 					</div>
-				</div>
-				<div class="form-group">
-					<label for="objspecselect">Data type</label>
-					<select id="objspecselect" class="form-element" disabled></select>
-				</div>
-				<div class="form-group">
-					<label>Number of rows <em class="text-muted">- optional</em></label>
-					<input id="nrows" type="text" class="form-element form-control" min="0" disabled>
-				</div>
-			</section>
-			<section class="form-section panel panel-default data-section">
-				<h2>Acquisition</h2>
-				<div class="form-group">
-					<label for="stationselect">Station</label>
-					<select id="stationselect" class="form-element" disabled></select>
-				</div>
-				<div class="form-group">
-					<label for="acqstartinput">From (UTC)</label>
-					<input id="acqstartinput" type="text" placeholder="YYYY-MM-dd'T'HH:mm:ss'Z'" class="form-element form-control" disabled>
-				</div>
-				<div class="form-group">
-					<label for="acqstopinput">To (UTC)</label>
-					<input id="acqstopinput" type="text" placeholder="YYYY-MM-dd'T'HH:mm:ss'Z'" class="form-element form-control" disabled>
-				</div>
-				<div class="form-group">
-					<label for="sampleheight">Sampling height (m) <em class="text-muted">- optional</em></label>
-					<input id="sampleheight" type="text" class="form-element form-control">
-				</div>
-				<div class="form-group">
-					<label for="instrumenturi">Instrument (URL) <em class="text-muted">- optional</em></label>
-					<input id="instrumenturi" type="text" class="form-element form-control">
-				</div>
-			</section>
-			<section class="form-section panel panel-default collection-section">
-				<h2>Collection</h2>
-				<div class="form-group">
-					<label for="collectionmembers">List of object urls (one per line)</label>
-					<textarea name="collectionmembers" id="collectionmembers" rows="8" cols="80" class="form-element"></textarea>
-				</div>
-				<div class="form-group">
-					<label for="collectiontitle">Title</label>
-					<input id="collectiontitle" type="text" placeholder="Collection title" class="form-element form-control">
-				</div>
-				<div class="form-group">
-					<label for="collectiondescription">Description</label>
-					<textarea name="collectiondescription" id="collectiondescription" rows="8" cols="80" class="form-element"></textarea>
-				</div>
-			</section>
+				</section>
+				<section class="col-md-4 data-section">
+					<div class="form-section panel panel-default">
+						<h2>Acquisition</h2>
+						<div class="form-group">
+							<label for="stationselect">Station</label>
+							<select id="stationselect" class="form-element" disabled></select>
+						</div>
+						<div class="form-group">
+							<label for="acqstartinput">From (@timezone)</label>
```

Comment:
This is probably a mistake. Time is expected to be provided in a format that includes time zone information, such as ISO-8601 with "Z" in the end.